### PR TITLE
chore(Storybook): update button stories

### PR DESCRIPTION
**Pull request by marcoskolodny**

### CODE REVIEW

File: src/__stories__/button-story.tsx [hunk 1]
```diff
@@ -1,370 +1,113 @@
...
-                    </Checkbox>
-                </Box>
-
-                {children}
-            </Box>
-        </ResponsiveLayout>
-    );
+const defaultArgs = {
+    text: 'Example',
+    loadingText: 'Loading text',
```
Comment (yceballost) [inline src/__stories__/button-story.tsx:28]: I would keep the loadingText empty to have the button with the most common behaviour in the story.

Comment (marcoskolodny) [reply src/__stories__/button-story.tsx]: done

File: src/__stories__/button-group-story.tsx [hunk 2]
```diff
@@ -10,51 +10,40 @@ export default {
 const handleOnPress = () => window.alert('pressed!');
 
 type Args = {
```
Comment (yceballost) [inline src/__stories__/button-group-story.tsx:12]: include new boolean control to change to small all the buttons, pls

**Review by yceballost — DISMISSED** [review]

File: src/__stories__/button-group-story.tsx [hunk 3]
```diff
@@ -10,51 +10,40 @@ export default {
...
+        <ButtonSecondary onPress={handleOnPress}>{secondaryButtonText}</ButtonSecondary>
+    ) : undefined;
+
+    const buttonLink = showLink ? <ButtonLink onPress={handleOnPress}>{linkText}</ButtonLink> : undefined;
+
+    return <ButtonGroup primaryButton={primaryButton} secondaryButton={secondaryButton} link={buttonLink} />;
 };
 
 Default.storyName = 'Button group';
 Default.args = {
```
Comment (atabel) [inline src/__stories__/button-group-story.tsx:43]: you can hide primaryButtonText/secondaryButtonText/linkText controls when showPrimaryButton/showSecondaryButton/showLink are false:
https://storybook.js.org/docs/react/api/arg-types#if

```ts
Default.argTypes = {
    primaryButtonText: {if: {arg: 'showPrimaryButton'}},
    secondaryButtonText: {if: {arg: 'showSecondaryButton'}},
    linkText: {if: {arg: 'showLink'}},
};
```

File: src/__stories__/button-story.tsx
Comment (atabel) [inline src/__stories__/button-story.tsx:1]: can we have the real component names here instead (ButtonPrimary, ButtonSecondary, etc)?
image

File: src/__stories__/button-group-story.tsx
[the remark this answers is not in the document]
Comment (yceballost) [reply src/__stories__/button-group-story.tsx:1]: Then we should mark this detail in storybook, right?

[the remark this answers is not in the document]
Comment (marcoskolodny) [reply src/__stories__/button-group-story.tsx:1]: Discussed offline. We don't see a good reason to mark it as internal in the storybook

File: src/__stories__/button-story.tsx
Comment (marcoskolodny) [reply src/__stories__/button-story.tsx:1]: fixed

File: src/__stories__/button-group-story.tsx (hunk 3 above)
Comment (marcoskolodny) [reply src/__stories__/button-group-story.tsx:56]: fixed

File: src/__stories__/button-group-story.tsx (hunk 2 above)
Comment (marcoskolodny) [reply src/__stories__/button-group-story.tsx:12]: done

**Review by yceballost — APPROVED** [review]

**Review by atabel — APPROVED** [review]

**Review by pladaria — APPROVED** [review]

**Comment by pladaria** [comment]
nice!

**Pull request merged**